From cf702e207b9518f59c2af3ebbb01cfffec014d37 Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Tue, 18 Aug 2020 16:11:23 +0800
Subject: [PATCH] ARM: dts: rockchip: add missing regulator settling time up
 property for rv1126 board

For PWM regulator, adjust voltage needs waiting time to be stable

Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
Change-Id: I8586ece1e109f3c8ab6a3f5da8ee4e298371ef01
---
 arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts | 2 ++
 arch/arm/boot/dts/rv1126-bat-ipc-v10.dts     | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts b/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
index 1c4fad3e2033..4b964970c2d2 100644
--- a/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
+++ b/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
@@ -61,6 +61,7 @@
 		regulator-init-microvolt = <825000>;
 		regulator-always-on;
 		regulator-boot-on;
+		regulator-settling-time-up-us = <250>;
 		pwm-supply = <&vcc3v3_sys>;
 		status = "okay";
 	};
@@ -78,6 +79,7 @@
 		regulator-init-microvolt = <825000>;
 		regulator-always-on;
 		regulator-boot-on;
+		regulator-settling-time-up-us = <250>;
 		pwm-supply = <&vcc3v3_sys>;
 		status = "okay";
 	};
diff --git a/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts b/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
index 26be3e5737a0..bc0795d55e5e 100644
--- a/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
+++ b/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
@@ -80,6 +80,7 @@
 		regulator-init-microvolt = <824000>;
 		regulator-always-on;
 		regulator-boot-on;
+		regulator-settling-time-up-us = <250>;
 		pwm-supply = <&vcc_3v3>;
 	};
 
@@ -92,6 +93,7 @@
 		regulator-init-microvolt = <824000>;
 		regulator-always-on;
 		regulator-boot-on;
+		regulator-settling-time-up-us = <250>;
 		pwm-supply = <&vcc_3v3>;
 	};
 };
-- 
2.35.3

